src/pages/Edit/ReviewEditPage.jsx

```diff
@@ -1,37 +1,67 @@
 import styled from 'styled-components';
 import { useState, useEffect } from 'react';
-import { useParams } from 'react-router-dom';
 import BackHeaderComponent from '../../components/common/BackHeaderComponent';
 import FundingParticipants from '../../components/FundingInfo/FundingParticipants';
 import ButtonComponent from '../../components/common/ButtonComponent';
 import BottomBackgroundComponent from '../../components/common/BottomBackgroundComponent';
 import { postReview, getReview, patchReview } from '../../api/review';
+import { getFundingInfo } from '../../api/funding';
+import { useNavigate, useParams } from 'react-router-dom';
 
 const ReviewEditPage = () => {
   const { fundingId } = useParams();
   const [reviewText, setReviewText] = useState('');
+  const [contributers, setContributers] = useState('');
+  const [isReviewExists, setIsReviewExists] = useState(false);
+  const [data, setData] = useState('');
+  const navigate = useNavigate();
+
   const handleInputChange = (e) => {
     setReviewText(e.target.value);
   };
-  const createReview = async (fundingId, reviewText) => {
+
+  const readFundingInfo = async () => {
+    try {
+      const res = await getFundingInfo(fundingId);
+      const data = res.data;
+      console.log(data);
+      data.contributers && setContributers(data.contributers);
+      if (data.existedReview === true) {
+        setIsReviewExists(true);
+      }
+      setData(data);
+    } catch (e) {
+      console.log(e);
+    }
+  };
+
+  const createReview = async () => {
     try {
       const response = await postReview(fundingId, reviewText);
       console.log(response.data);
     } catch (error) {
       console.error(error);
     }
   };
+
   const readReview = async (fundingId) => {
     try {
       const response = await getReview(fundingId);
-      if ((response.data.reviewContent = '')) setReviewText([]);
-      else setReviewText(response.data.reviewContent);
+      setReviewText(response.data.reviewContent);
       console.log(response.data);
     } catch (error) {
       console.error(error);
     }
   };
 
+  useEffect(() => {
+    const fetchReview = async () => {};
+    if (isReviewExists) {
+      readReview(fundingId);
+    }
+    fetchReview();
+  }, [fundingId]);
```

Comment:
이렇게 하면 더 간단하게 코드 작성할 수 있어요~
```suggestion
  useEffect(() => {
    isReviewExists && readReview(fundingId);
  }, [fundingId]);
```

Reply:
수정했습니다